language/diem-vm/mvhashmap/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "mvhashmap"
+version = "0.1.0"
+authors = ["Diem Association <opensource@diem.com>"]
+description = "Diem VM runtime"
+repository = "https://github.com/diem/diem"
+homepage = "https://diem.com"
+license = "Apache-2.0"
+publish = false
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+once_cell = "1.7.2"
```

Comment:
can't we use 1.8.0?

Reply:
I was just using the same version from the rest of the crates. Will that make a difference?

Reply:
I think this will work - but who makes sure we keep dependencies up to date over time?